src/modules/interp_met/WindNinja.cpp

```diff
@@ -241,15 +241,20 @@ void WindNinja::run(mesh& domain)
            if(W_transf>1.)
                W_transf = 1.+(Max_spdup-1.)*(W_transf-1.)/(transf_max-1.);
 
-           if (ninja_recirc){  // Need further test
+           if(compute_Sx){
+               if (ninja_recirc){  // Need further test
  
               //Compute what liston calls 'wind slope' using updated wind direction
-              double omega_s = face->slope() * cos(theta - face->aspect());
-              (*face)["omega_s"_s]= omega_s;
+            //  double omega_s = face->slope() * cos(theta - face->aspect());
```

Comment:
Can these comments be removed?

Reply:
I would prefer to keep it here as another option to correct wind on leeward slopes. Everything is really an early stage development